spec/integration/eirini_k8s_client_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'spec_helper'
+
+skip_eirini_tests = ENV['CF_RUN_EIRINI_SPECS'] != 'true'
```

Comment:
These tests don't appear to run in automation. If we add this var, they will fail without a kubeconfig. How are you testing this? We'd like to ensure test coverage, but it may not be appropriate to run it on the CC baseline.

Reply:
We will leave these tests turned off. We would recommend adding acceptance tests to https://github.com/cloudfoundry/capi-bara-tests to validate the CRDs are created and are correct. See [an example here](https://github.com/cloudfoundry/capi-bara-tests/blob/main/baras/route_crds.go). If these tests still provide value after adding to capi-bara-tests, we think it may be better to turn them into unit tests and mock out the actual k8s requests using webmock.